core/arch/proc_api.h

```diff
@@ -167,24 +167,25 @@ typedef struct {
  */
 #ifdef AARCHXX
 
-typedef enum {
-    AA64ISAR0,
-    AA64ISAR1,
-    AA64PFR0,
-    AA64MMFR1,
-    AA64DFR0,
-    AA64ZFR0,
-    AA64PFR1,
-    AA64ISAR2,
-    AA64_NUM_FEAT_REGS
-} feature_reg_idx_t;
-
 /**
- * For AArch64 this struct holds features registers' values read by MRS instructions.
- * Used by proc_get_all_feature_bits().
+ * For AArch64 the isa_features array holds features registers' values read by MRS
+ * instructions. Used by proc_has_feature().
  */
+typedef enum {
+    AA64ISAR0,         /**< ID_AA64ISAR0_EL1 */
```

Comment:
nit: Punctuate all Doxygen comments: so all of these here and below need a trailing `.`